src/schema/v2/ArtworkImport/artworkImport.ts

```diff
@@ -82,6 +82,9 @@ const ArtworkImportRowErrorType = new GraphQLObjectType({
     metadata: {
       type: GraphQLJSON,
     },
+    blocking: {
+      type: GraphQLBoolean,
```

Comment:
should this be non-nullable?

Reply:
Yes, great call out. Updated in [2a457bd](https://github.com/artsy/metaphysics/pull/6831/commits/2a457bd3403dcca276414062f0fae6bd81ceb416)